live/string/live.scm

```diff
@@ -13,6 +13,30 @@
 (define (string-blank? str)
   (string-every char-whitespace? str))
 
+(define (string->one-char-strings string)
+  (let loop ((strings '()) (n (string-length string)))
+    (if (zero? n) strings
+        (loop (cons (string-copy string (- n 1) n) strings) (- n 1)))))
+
+;; SRFI 140 (Immutable Strings) has the following:
+;;
+;; (string-split string delimiter [grammar limit start end]) -> list
+
+(define (string-split string delimiter)
```

Comment:
IMHO we can decide whether we want to add them or not. Do you think we should?

The most important thing is that the args that we do support, have the same meaning as in SRFIs.

This PR is just a subset of the functionality to get started with.

Reply:
I'm not sure if they are needed, as I see that was also not decided for R7RS-large yet, right? But since string-split with all arguments is quite widespread, maybe we should do that at some later point.

Reply:
Sure. Feel free to make a PR to add them if you want. There's probably code we can use with small changes in the sample implementation of SRFI 140.